src/05-objects.js

```diff
@@ -103,7 +118,10 @@ function addValueToObject(object, val) {}
  *  addKeyToObject({ right: false }, "left");
  *  //> { right: false, left: true }
  */
-function addKeyToObject(object, key) {}
+function addKeyToObject(object, key) {
+  object[key] = '420 hell yeah';
```

Comment:
While currently legal in the state of New York ( and should be everywhere ) it is worth remembering that references to cannabis and its related cultures may rub some recruiters the wrong way.  While highly, highly unlikely it is possible that someone could read this PR at some point.  I would suggest simply being aware of this fact before removing you having fun while coding :)

Reply:
I'm hoping the humor came through on your reading of this comment.  Fantastic use of the word 'transgressions'

Reply:
oh yeah 100% I laughed when you pointed it out.  I do see your point though!